app/helpers/location_helper.rb

```diff
@@ -134,10 +137,50 @@ def self.filter_by_name(samples_with_metadata, query)
   # See config/initializers/location_name_aliases.rb and add important known
   # aliases there.
   def self.normalize_name_aliases(name, geo_level)
```

Comment:
Would it make sense to rename this to `clean_location_name`? `normalize_name_aliases` is kind of confusing, and you're now doing an additional kind of cleaning.

Reply:
good idea, changed 👍

Reply:
Humm, I would not rename it... Lets not forget what is the real purpose of this function: It is to get a `normalized` name for each location to make sure that we do not have duplicates. 
It is not just cleaning...
If you do rename, please add a comment enforcing that goal.

Reply:
Hah, I like the `normalize` part, so how about just `normalize_location_name`

Reply:
Sure